jaeger-core/src/main/java/com/uber/jaeger/senders/InMemorySender.java

```diff
@@ -21,11 +21,13 @@
  */
 package com.uber.jaeger.senders;
```

Comment:
tbh, no idea why we need this guy. Can only imagine needing it to test RemoteReporter, but then it should be in tests-only.